internal/cloudapi/v2/handler.go

```diff
@@ -45,6 +45,19 @@ func (b binder) Bind(i interface{}, ctx echo.Context) error {
 	return nil
 }
 
+func (h *apiHandlers) GetVersion(ctx echo.Context) error {
+	spec, err := GetSwagger()
+	if err != nil {
+		return HTTPError(ErrorFailedToLoadOpenAPISpec)
+	}
+	version := Version{
+		Version:     spec.Info.Version,
```

Comment:
I'm sorry, but the API version is not what I expect under `Version` in the `/version` endpoint. Instead, I'd expect the release version. The API version is already included in the `/openapi` endpoint. Could you tell me what your reason is for picking the API version?

Reply:
I can rename this to APIVersion, is there any version information provided currently I could return in here?

Reply:
I don't really see anywhere where we could read the release version from code. Reading it from https://github.com/osbuild/osbuild-composer/blob/8b0a1d1714f66a72869d3b556ce756794aa19797/osbuild-composer.spec#L15 might be clunky at best. Is there a better way to read it from the system? :thinking: 

I think we could just omit the release for now and rename this field to APIVersion.

Reply:
No, we should not be reading it from the SPEC file or from the system :wink: 

It should be (and in most of the cases is) embedded in the binary itself, see https://github.com/osbuild/osbuild-composer/pull/4484#discussion_r1900581533

> I think we could just omit the release for now and rename this field to APIVersion.

In that case, I'd suggest entirely omitting the API version. What is the use case for listing it there?